tests/components/matter/test_binary_sensor.py

```diff
@@ -414,3 +414,24 @@ async def test_thermostat_occupancy(
     state = hass.states.get("binary_sensor.longan_link_hvac_occupancy")
     assert state
     assert state.state == "off"
+
+
+@pytest.mark.parametrize("node_fixture", ["eve_shutter"])
+async def test_shutter(
```

Comment:
We may want to include what we're testing in the test name as well:
```suggestion
async def test_shutter_problem(
```